packages/runtime/src/useCases/transport/messages/SendMessage.ts

```diff
@@ -73,21 +74,48 @@ export class SendMessageUseCase extends UseCase<SendMessageRequest, MessageDTO>
             );
         }
 
-        if (!(transformedContent instanceof Request)) return;
+        if (transformedContent instanceof Notification) return;
 
-        if (!transformedContent.id) return RuntimeErrors.general.invalidPropertyValue("The Request must have an id.");
+        const deletedPeers: string[] = [];
+        const peersInDeletion: string[] = [];
+        const peersWithNoActiveRelationship: string[] = [];
+        for (const recipient of recipients) {
+            const relationship = await this.relationshipsController.getActiveRelationshipToIdentity(CoreAddress.from(recipient));
+            if (!relationship) {
+                peersWithNoActiveRelationship.push(recipient);
+                continue;
+            }
+            if (relationship.peerDeletionInfo?.deletionStatus === PeerDeletionStatus.Deleted) {
+                deletedPeers.push(recipient);
+            }
+            if (relationship.peerDeletionInfo?.deletionStatus === PeerDeletionStatus.ToBeDeleted) {
+                peersInDeletion.push(recipient);
+            }
```

Comment:
Since you'll be touching the code again anyways, please add blank lines between the if-statements for better readability.